presto-main/src/main/java/io/prestosql/server/QuerySessionSupplier.java

```diff
@@ -18,6 +18,7 @@
 import io.prestosql.security.AccessControl;
```

Comment:
- what is the motivation (use case) behind this?
 - don't you want to provide a session property for that?

Reply:
> what is the motivation (use case) behind this?

Surfaced on Slack as a user's request.

> don't you want to provide a session property for that?
 
The point is not to have update clients and still force certain zone. 
If updating clients is an option, they would be simply configured to send the desired zone themselves.

So no, I don't plan to provide a session property for that.